apps/passport-client/.env.example

```diff
@@ -41,4 +41,9 @@ GPC_ARTIFACTS_CONFIG_OVERRIDE=
 # GPC_ARTIFACTS_CONFIG_OVERRIDE={"source": "unpkg", "stability": "test", "version": "0.0.2"}
 
 # Use this value to enable one-click login. Set to "true" to enable. Set to "false", or don't set at all to disable.
-# ONE_CLICK_LOGIN_ENABLED=true
\ No newline at end of file
+# ONE_CLICK_LOGIN_ENABLED=true
+
+# If Zapp POD signing is to be restricted, set this to "true".
+ZAPP_RESTRICT_ORIGINS=true
+# If Zapp POD signing is to be restricted, set this to the allowed origins, separated by commas.
+ZAPP_ALLOWED_SIGNER_ORIGINS="http://example.com,http://localhost:3200"
```

Comment:
nit, but noting that our convention is typically to have a stringified array of strings, e.g. with `FROGCRYPTO_ADMIN_USER_EMAILS` or `GENERIC_ISSUANCE_PUBLIC_KEY`

Reply:
I've changed this now!